
From: Anton Blanchard <anton@samba.org>

Fix another gcc 3.5 compile issue, this time the default_policy prototype
was not marked static whereas the definition was. There is no need for
the prototype, so remove it.

Signed-off-by: Anton Blanchard <anton@samba.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/mempolicy.h |    3 ---
 1 files changed, 3 deletions(-)

diff -puN include/linux/mempolicy.h~fix-gcc-35-compile-issue-in-mm-mempolicyc include/linux/mempolicy.h
--- 25/include/linux/mempolicy.h~fix-gcc-35-compile-issue-in-mm-mempolicyc	Wed Aug  4 16:48:50 2004
+++ 25-akpm/include/linux/mempolicy.h	Wed Aug  4 16:48:50 2004
@@ -68,9 +68,6 @@ struct mempolicy {
 	} v;
 };
 
-/* An NULL mempolicy pointer is a synonym of &default_policy. */
-extern struct mempolicy default_policy;
-
 /*
  * Support for managing mempolicy data objects (clone, copy, destroy)
  * The default fast path of a NULL MPOL_DEFAULT policy is always inlined.
_
